pkg/stable.packages.props

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <!-- NETCore 1.0 core-setup -->
+    <StablePackage Include="Microsoft.NETCore.App">
+      <Version>1.0.0</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore">
+      <Version>5.0.1</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.UniversalWindowsPlatform">
+      <Version>5.2.0</Version>
+    </StablePackage>
+    <StablePackage Include="Microsoft.NETCore.DotNetHost">
+      <Version>1.0.1</Version>
```

Comment:
It isn't part of the build.  It's an dumb tool that I wrote a year or so ago around the first time we did this for UWP.